src/js/controllers/walletHome.js

```diff
@@ -28,6 +28,27 @@ angular.module('copayApp.controllers').controller('walletHomeController', functi
   this.addr = {};
   this.lockedCurrentFeePerKb = null;
 
+  self.hideBalance = function() {
+    var fc = profileService.focusedClient;
+    storageService.getHideBalanceFlag(fc.credentials.walletId, function(err, shouldHideBalance) {
+      if (err) self.shouldHideBalance = false;
+      else self.shouldHideBalance = (shouldHideBalance == 'true') ? true : false;
```

Comment:
should be like https://github.com/bitpay/copay/pull/3872/files#diff-313c93846dfce71c19438c4889833d7fR43

no need to check against  string 'true'